t/ui/PhantomTest.pm

```diff
@@ -120,15 +119,22 @@ sub make_screenshot($) {
     close($fh);
 }
 
-sub call_phantom {
-    my ($schema_hook) = @_;
-    # fail if phantomjs or Selenium::Remote::Driver are unavailable
-    use IPC::Cmd qw[can_run];
+sub check_phantom_modules {
+    # load required modules if possible. DO NOT EVER PUT THESE IN
+    # 'use' FUNCTION CALLS! Always use can_load! Otherwise you will
+    # break the case where they are not available and tests should
+    # be skipped.
     use Module::Load::Conditional qw(can_load);
-    if (!can_load(modules => {'Selenium::PhantomJS' => undef,})) {
-        return undef;
-    }
+    my $modname = $ENV{SELENIUM_CHROME} ? 'Test::Selenium::Chrome' : 'Test::Selenium::PhantomJS';
+    my $modver = $ENV{SELENIUM_CHROME} ? '1.02' : undef;
```

Comment:
we need 1.02 for both. I just specified it once as they come from the same module.